scenes/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,31 @@
+import {AppPage} from './app.po';
+import {screenshot} from '../screenshot';
+
+describe('workspace-project App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('screenshot for input scene', () => {
```

Comment:
I would just make a list `['input', 'button', ...]` and then you can generate the tests in a for loop